selenium-system-test/src/test/resources/features/airGapTests/Air-gap tests for testing with NS&I.feature

```diff
@@ -0,0 +1,117 @@
+      | postcode       | BN435QPABC                                                                                                                                                                                                                                                                                                   |
+      | country code   | 01                                                                                                                                                                                                                                                                                                           |
+
+    When they choose to go ahead with creating an account
+    Then they see that the account is created
+
+
+  Scenario: Air-gap 4 - All fields are populated with values of minimum length
+    Given an applicant has the following details:
+      | field          | value      |
+      | first name     | A          |
+      | last name      | 4          |
+      | NINO           | <eligible> |
+      | date of birth  | 12/12/1999 |
+      | email address  | a@a        |
+      | address line 1 | a          |
+      | address line 2 | b          |
+      | address line 3 | c          |
+      | address line 4 | d          |
+      | address line 5 | e          |
+      | postcode       | B          |
+      | country code   | 01         |
+
+    When they choose to go ahead with creating an account
+    Then they see that the account is created
+
+
+  Scenario: Air-gap 7 - Wacky postal address
+    Given an applicant has the following details:
+      | field          | value                    |
+      | first name     | AirGap                   |
+      | last name      | Seven                    |
+      | NINO           | <eligible>               |
+      | date of birth  | 12/12/1999               |
+      | email address  | sarah@smith.com          |
+      | address line 1 | C/O Fish 'n' Chips Ltd.  |
+      | address line 2 | The Tate & Lyle Building |
+      | address line 3 | Carisbrooke Rd.          |
+      | address line 4 | Barton-under-Needwood    |
+      | address line 5 | Derbyshire               |
+      | postcode       | W1J7NT                   |
+      | country code   | 04                       |
+
+    When they choose to go ahead with creating an account
+    Then they see that the account is created
+
+
+  Scenario: Air-gap 11 - Populate forename, surname, address lines (phone number) and email with accented characters
+    Given an applicant has the following details:
+      | field          | value                    |
+      | first name     | AirGapRené Chloë         |
+      | last name      | ElevenO'Connor-Jørgensen |
+      | NINO           | <eligible>               |
```

Comment:
why only `<eligible>` in all scenarios ?

Reply:
I wanted to highlight that the user is eligible for HtS. At present, only "eligible" exists as a tag in the feature files to describe a NINO belonging to an eligible user - but I plan to introduce other tags.

Reply:
I'm going to merge this in, we can add more scenarios later